cmd/symbols/squirrel/lang_starlark.go

```diff
@@ -0,0 +1,184 @@
+package squirrel
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	sitter "github.com/smacker/go-tree-sitter"
+
+	"github.com/sourcegraph/sourcegraph/internal/types"
+	"github.com/sourcegraph/sourcegraph/lib/errors"
+)
+
+func (squirrel *SquirrelService) getDefStarlark(ctx context.Context, node Node) (ret *Node, err error) {
+	defer squirrel.onCall(node, String(node.Type()), lazyNodeStringer(&ret))()
+	switch node.Type() {
+	case "identifier":
+		return squirrel.getDefStarlarkIdentifier(ctx, node)
+	case "string":
+		return squirrel.getDefStarlarkString(ctx, node)
+	default:
+		return nil, nil
+
+	}
+}
+
+func (squirrel *SquirrelService) getDefStarlarkIdentifier(ctx context.Context, node Node) (ret *Node, err error) {
+	symbol := node.Node.Content(node.Contents)
+	root := Node{
+		RepoCommitPath: node.RepoCommitPath,
+		Node:           getRoot(node.Node),
+		Contents:       node.Contents,
+		LangSpec:       node.LangSpec,
+	}
+	exports, err := starlarkBindingNamed(symbol, root)
+	if err != nil {
+		return nil, err
+	}
+	if exports != nil {
+		return &Node{
+			RepoCommitPath: node.RepoCommitPath,
+			Node:           exports,
+			Contents:       node.Contents,
+			LangSpec:       node.LangSpec,
+		}, nil
+	}
+	fmt.Println(node)
+	return nil, nil
```

Comment:
```suggestion
	exports, err := starlarkBindingNamed(symbol, root)
	if err != nil || exports == nil {
		return nil, err
	}
	return &Node{
		RepoCommitPath: node.RepoCommitPath,
		Node:           exports,
		Contents:       node.Contents,
		LangSpec:       node.LangSpec,
	}, nil
```

I believe its functionally equivalent, and removes the `fmt.Println` :P